src/web/views/enter-name.njk

```diff
@@ -12,7 +13,8 @@
     }
   }) %}
 
-    <p class="govuk-body">{{ formDescription }}</p>
+    <br>
+    <span class="govuk-hint">{{ formDescription }}</span>
```

Comment:
`{{ formDescription }}` doesn't seem like the correct variable name. Would `{{ hint }}` or `{{ nameFieldHint }}` be better?

Reply:
Updated to hint.